app/components/notifications-page/message/index.ts

```
@@ -44,10 +47,13 @@ export default class NotificationsPageMessageComponent extends Component<Notific
 
   get notificatonContext() {
     const contextClass = this.notificationMapObject.context;
+
     if (!contextClass) {
-      this.logger.error(`class not defiend for ${this.componentName}`);
+      this.logger.error(`class not defined for ${this.componentName}`);
+
       return;
```

Biome (1.9.4)

[error] This return should return a value because it is located in a getter.



(lint/suspicious/useGetterReturn)